src/client/leihs/borrow/features/about_page/core.cljs

```diff
@@ -41,7 +42,7 @@
 
 (defn view []
   [:section.m-3
-   [:h1.text-xl.font-black "ABOUT"]
+   [:h1.text-xl.font-black (t :borrow.about-page/title)]
```

Comment:
Is it possible to have this nesting under the `:borrow` key, but not have to spell it out? 
Something like wrapping the function and settings this as a default.

Could be global (in all of borrow) but to configure it once per file/namespace import.
Even better would be if we could leave out the "feature scope" as well.
Those 2 wishes could be related or not. 

```cljs
; maybe start of namespace
(translations-set-defaults :borrow.about-page)
; inside the ui components I just want to write
(t :title) ; gets (t :borrow.about-page/title)
; but one needs also to get translations from other "scopes"
(t :shopping-cart/timeout-msg) ;  (t :borrow.shopping-cart/timeout-msg)
(t :shared.actions-names/button-submit) ; just like that, its outside of `:borrow`
```

Reply:
wish nr. 1 -> easy
wish nr. 2 -> should be possible, I will try. Let me first merge the initial setup.

Reply:
@eins78 and sometimes we can have sub-maps inside the default scope (`baz`). how to differentiate between `(t :other-ns/foo)` and `(baz/foo)`?

Reply:
I would maybe suggest `:.foo/baz` ?

Reply:
adapted my examples per our discussion:

```clj
; maybe start of namespace
(translations-set-defaults :borrow.about-page)
; inside the ui components I just want to write
(t :title) ; gets (t :borrow.about-page/title)
; but one needs also to get translations from other "scopes"
(t :!borrow.shopping-cart/timeout-msg) ;  (t :borrow.shopping-cart/timeout-msg)
(t :!shared.actions-names/button-submit) ; just like that, its outside of `:borrow`
```